zivid_samples/launch/zivid_camera_with_serial_number.launch

```diff
@@ -0,0 +1,6 @@
+<launch>
+  <arg name="serial_number" default="X"/>
```

Comment:
set X to SERIAL?

Reply:
Maybe better to remove the default? It makes little sense to invoke this without the serial number. If I do this, I get the following:

```
> ros2 launch zivid_samples zivid_camera_with_serial_number.launch
[INFO] [launch]: All log files can be found below C:\Users\MichaelRPRagazzon\.ros\log\2024-07-22-16-10-01-579606-DESKTOP-3CHQG3M-18084
[INFO] [launch]: Default logging verbosity is set to INFO
[ERROR] [launch]: Caught exception in launch (see debug for traceback): Included launch description missing required argument 'serial_number' (description: 'no description given'), given: []
```

If we add a description too, e.g. `<arg name="serial_number" description="Serial number of camera to connect to"/>`, we get this output:

```
[ERROR] [launch]: Caught exception in launch (see debug for traceback): Included launch description missing required argument 'serial_number' (description: 'Serial number of camera to connect to'), given: []
```

Reply:
f43ee1192b8df8861a4394c90bae795000decff9